server/integration-tests/using-different-ports.sh

```diff
@@ -1,4 +1,4 @@
-#/bin/bash
+#!/bin/bash
```

Comment:
Nice fix!

Reply:
It was actually a handy code inspection in Intellij that pointed this out.